utils/field.h

```diff
@@ -39,12 +39,16 @@ enum display_field_id {
 	REPORT_F_TOTAL_TIME = 0,
 	REPORT_F_TOTAL_TIME_AVG,
 	REPORT_F_TOTAL_TIME_MIN,
+	REPORT_F_TOTAL_TIME_MIN_TS,
 	REPORT_F_TOTAL_TIME_MAX,
+	REPORT_F_TOTAL_TIME_MAX_TS,
 
 	REPORT_F_SELF_TIME,
 	REPORT_F_SELF_TIME_AVG,
 	REPORT_F_SELF_TIME_MIN,
+	REPORT_F_SELF_TIME_MIN_TS,
 	REPORT_F_SELF_TIME_MAX,
+	REPORT_F_SELF_TIME_MAX_TS,
```

Comment:
Adding new fields in the middle created some problems.  Let's add them after the STDV fields.